### PR TITLE
fix tagging

### PR DESCRIPTION
**Pull request by himank**

### CODE REVIEW

File: server/v1/api.proto [hunk 1]
```diff
@@ -45,27 +45,18 @@ option (openapi.v3.document) = {
     name: "Authentication"
     description: "The auth section of API provides open authorization (OAuth 2.0) APIs. Tigris supports pluggable OAuth provider. Pass the token in the headers for authentication, as an example `-H \"Authorization: Bearer eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6I\"`(replace it with your token). All API requests must be made over HTTPS. Calls made over plain HTTP will fail. API requests without authentication will also fail."
   },{
-    name: "Namespaces"
-    description: "The Namespaces section provide APIs that can be used to manage namespaces. A single namespace can have one or more projects."
+    name: "Application Keys"
+    description: "The application Keys section provide APIs that can be used to manage keys for your project. A single project can have one or more keys."
```
Comment (ovaistariq) [inline server/v1/api.proto:49]: manage application keys
one or more application keys

File: server/v1/openapi.yaml [hunk 2]
```diff
@@ -3175,27 +3175,21 @@ components:
 security:
     - BearerAuth: []
 tags:
+    - name: Application Keys
+      description: The application Keys section provide APIs that can be used to manage keys for your project. A single project can have one or more keys.
```
Comment (ovaistariq) [inline server/v1/openapi.yaml:3179]: application keys section

File: server/v1/openapi.yaml [hunk 3]
```diff
@@ -3175,27 +3175,21 @@ components:
...
     - name: Authentication
       description: 'The auth section of API provides open authorization (OAuth 2.0) APIs. Tigris supports pluggable OAuth provider. Pass the token in the headers for authentication, as an example `-H "Authorization: Bearer eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6I"`(replace it with your token). All API requests must be made over HTTPS. Calls made over plain HTTP will fail. API requests without authentication will also fail.'
```
Comment (ovaistariq) [inline server/v1/openapi.yaml:3181]: provides OAuth 2.0 APIs

File: server/v1/openapi.yaml [hunk 4]
```diff
@@ -3175,27 +3175,21 @@ components:
...
     - name: Collections
       description: The Collections section provide APIs that can be used to manage collections. A collection can have one or more documents.
     - name: Databases
-      description: The Databases section provide APIs that can be used to manage Tigris Databases. A single Database can have one or more collections.
-    - name: Documents
-      description: The Documents section provide APIs to manage documents insides a collection. Uniqueness of documents are enforced by the primary key.
-    - name: Event Streaming
-      description: Tigris has built-in event streaming support. This functionality is enabled for the `Messages` collection type. This type of collection is referred as Topic and has endpoints that allows you to both subscribe and publish to a topic which enable building event-driven applications. Ordering is guaranteed on a partition level
+      description: The Databases section provide APIs that can be used to manage Tigris Databases. A single Database can have one or more collections. A database is created automatically for you when you create a project.
```
Comment (ovaistariq) [inline server/v1/openapi.yaml:3187]: used to interact with the databases

**Review by efirs — APPROVED** [review]

**Pull request merged**